src/ctia/schemas/core.clj

```diff
@@ -11,6 +11,26 @@
             [schema-tools.core :as st]
             [schema.core :as s :refer [Bool Str]]))
 
+(s/defschema APIHandlerServices
+  "Maps of services available to routes"
+  {:ConfigService {:get-config (s/=> s/Any s/Any)
+                   :get-in-config (s/=> s/Any
+                                        [s/Any]
+                                        [s/Any s/Any])}
+   :HooksService {:apply-hooks (s/pred ifn?) ;;keyword varargs
+                  :apply-event-hooks (s/=> s/Any s/Any)}
+   :StoreService {:read-store (s/pred ifn?) ;;varags
+                  :write-store (s/pred ifn?)} ;;varags
```

Comment:
you probably need more keys here: https://github.com/threatgrid/ctia/blob/master/src/ctia/store.clj

Reply:
Turns out the routes only use these two.